aws/resource_aws_rds_cluster.go

```diff
@@ -96,9 +96,18 @@ func resourceAwsRDSCluster() *schema.Resource {
 			},
 
 			"engine": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "aurora",
+				ForceNew:     true,
+				ValidateFunc: validateRdsEngine,
+			},
+
+			"engine_version": {
```

Comment:
We should also read the engine version around line 575

Reply:
right, missed that one.